packages/flutter_adaptive_scaffold/lib/src/breakpoints.dart

```diff
@@ -294,11 +294,10 @@ class Breakpoint {
         ? width >= lowerBoundWidth
         : width >= lowerBoundWidth && width < upperBoundWidth;
 
-    final bool isHeightActive = isDesktop ||
-        orientation == Orientation.portrait ||
-        (orientation == Orientation.landscape && andUp
+    final bool isHeightActive =
```

Comment:
Can this just be `isPortrait || isWidthActive || andUp`?

Reply:
I'm in doubt how to properly handle all cases. According to the specs it should check height, but it is hard to handle edge cases. The new tests cover a lot of it.

Reply:
Sorry, I didn't mean the entire case should be only that, just the first line. Because if `isPortrait` is false then the first case will fail and check the second case, I don't think we have to check if `isPortrait` is false in the second case. Or would this fail?

Reply:
Aah right. Fixed now.